### PR TITLE
Console app

### PR DESCRIPTION
**Pull request by SekoViper**
# Summary
## Implement the following

1. Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following:

- List all books.
- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.
- List all rentals for a given person id.

2. In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:

- Present the user with a list of options to perform.
- Lets users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Seko,
 
 While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
 
 ## Invalid Code Review Request
You have submitted a project, whereby rubocop linter checks are failing on your Pull Request. Kindly ensure that **all** checks are passing and request another code review.
 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _Invalid Code Review Request **does not count** into the code reviews limit._

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    puts 'What type of person do you want to create? Enter 1 for teacher or 2 student'
+    person_type = gets.chomp
+    case person_type
+    when 1
+      puts 'Enter Teachers Name'
+      name = gets.chomp
+      puts "Enter teacher's age"
+      age = gets.chomp.to_i
+      puts "Waht's teacher's specialization?"
+      specialization = gets.chomp
```
Comment (BertrandConxy) [inline main.rb]: - [ ] Good job so far 👏 When I try to run your program, ruby is flagging the syntax errors in the `main.rb` file. I tried to debug it, but every time it is showing another error. Please verify that it is working well and if you found out that something is wrong like syntax errors, please correct it. It is better if you could add procedures to run your program if they are specific in the README file. Here is the screenshot I took 👇 
image

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @SekoViper, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @SekoViper,

Good job working on the changes highlighted by the previous reviewer 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ App UI is created correctly 👍🏽
✅ Good descriptive PR 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**